enclave/core/CMakeLists.txt

```diff
@@ -134,6 +134,14 @@ target_compile_options(oecore PUBLIC
     -fno-stack-protector
     -fvisibility=hidden)
 
+# According to https://gcc.gnu.org/onlinedocs/gcc/Code-Gen-Options.html#Code-Gen-Options
+# "The default without -fpic is ‘initial-exec’; with -fpic the default is ‘global-dynamic’"
+# Enclaves are linked using -pie and therefore global-dynamic is too conservative.
+# Of the two efficient static tls models, inital-exec and local-exec, we choose the most
+# optimal one.
+target_compile_options(oecore PUBLIC
```

Comment:
This is jsut for style's sake, but could you merge this with the above `target_compile_options`? You can (and should) keep the comment inline, but I don't like having so many instances of `target_compile_options`, it gets confusing to readers to deduce just exactly where they all are.